pkg/stack/stack.go

```diff
@@ -23,29 +27,55 @@ type Stack struct {
 	Description string `json:"description" yaml:"description"`
 }
 
+// StacksIndexURL to get index
+const StacksIndexURL = "https://stack.h8r.io/index"
+
 var (
 	// ErrNotExist mean this stack doesn't exist.
 	ErrNotExist = errors.New("target stack doesn't exist")
 )
 
-// Stacks stores all available stacks.
-var Stacks = map[string]struct{}{
-	"sample":     {},
-	"gin-next":   {},
-	"spring-vue": {},
-	"gin-vue":    {},
+// List all stacks
+func List() ([]Stack, error) {
+	type Index struct {
```

Comment:
It is better to define type outside the function.  
or you can use anonymous struct.

Reply:
+1

Reply:
Using anonymous struct now